src/applications/personalization/dashboard-2/components/health-care/HealthCare.jsx

```diff
@@ -69,65 +58,58 @@ const HealthCare = ({
     );
   }
 
+  const messagesText =
+    typeof unreadMessagesCount === 'number'
+      ? `You have ${unreadMessagesCount} new messages`
```

Comment:
This is logic as a result of a design update - even if the number comes back as '0', we still show that they have 0 messages.

I will revisit this logic while working on [this ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/21849)

![image](https://user-images.githubusercontent.com/14869324/112036998-e1d79a00-8b06-11eb-8a42-4b293ce2d3df.png)

Reply:
`messages` will need to be `message` if the `unreadMessagesCount === 1`